frontend/src/components/Search.tsx

```diff
@@ -1,17 +1,22 @@
 import { debounce } from 'lodash'
 import { Search, X } from 'lucide-react'
-import React, { useCallback, useRef, useState } from 'react'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
 
 interface SearchProps {
-  // eslint-disable-next-line no-unused-vars
+  /* eslint-disable-next-line */
```

Comment:
Why this disabled ? ....rest all is good 💯

Reply:
onSearch: (query: string) => void  because of query being unused in this